feat: added docker-compose (#101)
gis-ops/valhalla-app: README.md

> # Valhalla

Reviewer: Valhalla is here: https://github.com/valhalla/valhalla. This is the demo web app. If you really want to, you can scratch the "Demo"s

This is the ReactJS demo web app running on https://valhalla.openstreetmap.de. It provides routing and isochrones with a magnitude of options and makes requests to [Valhalla](https://github.com/valhalla/valhalla), an open source routing engine and accompanying libraries for use with OpenStreetMap data.

![valhalla_screenshot](https://user-images.githubusercontent.com/10322094/144841673-18ec0772-129d-443e-a040-5172480b0f92.png)

## 🛠️ Development

Using one of the following ways you can contribute to the repo;

1. Local development
2. Local development with Docker Compose

> ### Local development

Reviewer: I think this is redundant, dev is always "local" dev, no?

#### Prerequisites

- [NodeJs](https://nodejs.org/en/download/)

#### Commands

You can set this up locally with the following steps:

1. `npm install`
2. `npm run start`

You can now access the app on http://localhost:3000

### Local development with Docker Compose

> This will allow you to run your favourite IDE but not have to install any dependencies on your computer like NodeJS.

Reviewer: ```suggestion
This will allow you to run your favourite IDE without having to install any dependencies on your computer like NodeJS.
```
